README.md

```diff
@@ -55,5 +56,7 @@ openstack-ansible setup-everything.yml
 # Ansible Roles
 
 * `horizon_extensions`
+* `logstash`
+* `module_import`
```

Comment:
It's since been removed from the master branch and this line is not part of the PR.